include/fmt/format.h

```diff
@@ -967,10 +994,27 @@ inline int count_digits(uint64_t n) {
     count += 4;
   }
 }
+
+#if defined(FMT_BUILTIN_CLZLL)
```

Comment:
Please avoid unnecessary changes.

Reply:
I'm sorry but this is necessary. Maybe there are not enough tests for that (like test with `uint32_t` and `uint64_t` values), but I can add them. It's just unable to work without `constexpr` `count_digits`.

Reply:
Actually, I _will_ add them anyway.

Reply:
Test with uint32_t and uint64_t values added.

Reply:
Changing `#ifdef` to `#if defined(FMT_BUILTIN_CLZLL)` is unnecessary. Same elsewhere.

Reply:
Ok, got it